source/compute/mus_compute_bgk_module.fpp

```diff
@@ -61,9 +61,11 @@ module mus_bgk_module
 
   private
 
-  public :: mus_advRel_kCFD_rBGK_vStdNoOpt_l
-  public :: bgk_advRel_flekkoy
   public :: bgk_advRel_flekkoy_noFluid
+  public :: mus_advRel_kCFD_rBGK_vStdNoOpt_l
+  public :: mus_advRel_kPS_rBGK_v1st_l
+  public :: mus_advRel_kPS_rBGK_v2nd_l
+  public :: mus_advRel_kPS_rTRT_vStdNoOpt_l
```

Comment:
Could you please move all passive scalar routines to seperate module `mus_compute_passiveScalar_module.fpp`

Reply:
The code has been moved to a new file named `mus_compute_passiveScalar_module.fpp`.